packages/flutter/test/material/text_field_test.dart

```diff
@@ -756,7 +756,7 @@ void main() {
     await tester.pump();
 
     expect(controller.selection.baseOffset, selection.baseOffset);
-    expect(controller.selection.extentOffset, selection.extentOffset);
+    expect(controller.selection.extentOffset, selection.extentOffset + 2);
```

Comment:
tott/510.md

Don't do redirections, just put numbers in, e.g. base 4, extent 9 directly.